ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixParticipant.java

```diff
@@ -176,33 +236,37 @@ public Task createNewTask(TaskCallbackContext context) {
   }
 
   /**
-   * Get the list of sealed replicas from the HelixAdmin. This method is called only after the helixAdministrationLock
+   * Set the list of sealed replicas in the HelixAdmin. This method is called only after the helixAdministrationLock
    * is taken.
-   * @return list of sealed replicas from HelixAdmin
+   * @param sealedReplicas list of sealed replicas to be set in the HelixAdmin
+   * @return whether the operation succeeded or not
    */
-  private List<String> getSealedReplicas() {
+  private boolean setSealedReplicas(List<String> sealedReplicas) {
     InstanceConfig instanceConfig = helixAdmin.getInstanceConfig(clusterName, instanceName);
     if (instanceConfig == null) {
       throw new IllegalStateException(
           "No instance config found for cluster: \"" + clusterName + "\", instance: \"" + instanceName + "\"");
     }
-    return ClusterMapUtils.getSealedReplicas(instanceConfig);
+    logger.trace("Setting InstanceConfig with list of sealed replicas: {}", Arrays.toString(sealedReplicas.toArray()));
```

Comment:
you don't need the `Arrays.toString(sealedReplicas.toArray())`

Reply:
removed.